### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by Leboroz**
## Features Added: ♻️ 

- I removed all hardcoded items from the tasks array.
- I created a new JavaScript file for the new functionality.
- I Implemented a function for adding a new task.
- I Implemented a function for deleting a task.
- I Implemented a function for editing task descriptions.

### CODE REVIEW

File: src/sass/index.scss [hunk 1]
```diff
@@ -1,51 +1,22 @@
...
+
+/* -------------------------------------------------------------------------- */
+@import "./custom";
+
+/* -------------------------------------------------------------------------- */
+
+/*                             //IMPORT COMPONENTS                            */
+
+/* -------------------------------------------------------------------------- */
+@import "./components/mixins";
```
Comment (D4nielJ) [inline src/sass/index.scss]: - [x] It seems like there are compiling problems: I'm getting the following error on my local environment. Can you make sure to include `./components/mixins` in the repository or remove it from this file if it is empty?

image

Comment (D4nielJ) [reply src/sass/index.scss]: I removed the line 13 assuming mixins is an empty file that didn't get pushed to GitHub, but then I get this error: 
image

Make sure to fix all of these issues so the next reviewer can perform a review without compiling errors. I recommend you that if extra steps need to run the project locally you include these on the README file.

File: src/components/todoitem.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default class TodoItem {
+  constructor(description, completed = false) {
```
Comment (D4nielJ) [inline src/components/todoitem.js]: - [x] Let's make sure that when a to-do list is marked as completed all the changes are persisted on the LocalStorage as well.

no-completed

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @Leboroz, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: 
- I can add, remove, and remove completed todos!
- Most of the changes are persisted on LocalStorage

## Required Changes ♻️

- [ ] I think it would be good if you customize the README file with all of the information about the project. This is important for several reasons, from keeping the repository professional and well documented to make sure that people wanting to collaborate can run correctly the project on their local machines.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leolpaz — APPROVED** [review]
Hi @Leboroz , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**